src/plugins/initiatives/initiativeFile.test.js

```diff
@@ -13,7 +14,7 @@ import {
 
 const exampleInitiativeFile = (): InitiativeFile => ({
   title: "Sample initiative",
-  timestampIso: ("2020-01-08T22:01:57.766Z": any),
+  timestampIso: Timestamp.toISO(Date.parse("2020-01-08T22:01:57.766Z")),
```

Comment:
Note: as I found this, I considered adding another validationg function like `(string) => TimestampISO`. But for tests this is actually a validated pattern. And not having this function discourages using ISO's as the internal representation.

`Date.parse(string)` would return `NaN` if the timestamp is invalid. And `Timestamp.toISO` would throw because `Number.isInteger(NaN) === false`.